server/methods/core/shipping.js

```diff
@@ -150,6 +190,37 @@ function updateShippingRecordByShop(cart, rates) {
       Logger.debug(`Success updating rates for cart ${cartId}`, rates);
     });
   });
+  pruneShippingRecordsByShop(cart);
+  normalizeAddresses(cart);
+}
+
+function getDefaultAddress(cart) {
+  const userId = cart.userId;
+  const account = Accounts.findOne(userId);
+  if (account && account.profile && account.profile.addressBook) {
+    const address = account.profile.addressBook.find((addressEntry) => addressEntry.isShippingDefault === true);
```

Comment:
Are we guaranteed to have an address with `isShippingDefault`, that's true from my recollection in which case 👍

Reply:
AFAIK if we have an addressBook we always have a default shipping address